src/session.lisp

```diff
@@ -570,7 +570,8 @@
                                                             :stderr-fn (lambda (data)
                                                                            (when (assoc :show-stderr params)
                                                                                  (send-msg state (notification:stderr data)))))))
-
+                                     ;  (send-msg state (message:create-response id :result-value T))
+                                     (send-msg state (resp:load-file id msgs))
```

Comment:
It needs to just be this line. resp:load-file creates the load file response message. Apparently when I moved this to run in a thread, I forgot to wrap that in send-msg so it would send the response. The handlers that don't run in a thread return their responses, which get sent in read-messages.